apiary.apib

```diff
@@ -457,7 +939,7 @@ Fetches asset with provided assetId in the system.
     + Body
 
 ### /assets{?perPage,page,createdBy,fromTimestamp,toTimestamp,identifier}
```

Comment:
Seems that there's no such endpoint

Reply:
There is an endpoint to use with Atlas. This doc is for both Hermes and Atlas